src/main/resources/assets/tfc/recipes/powderkeg.json

```diff
@@ -0,0 +1,17 @@
+{
+  "type": "minecraft:crafting_shapeless",
+  "ingredients": [
+    {
+      "type": "forge:ore_dict",
+      "ore": "barrel"
+    },
+    {
+      "item": "minecraft:gunpowder"
+    }
```

Comment:
This should also include red dye. Why? The texture is red and this is a really boring recipe.

Reply:
Great input, and something I'd considered. 
I'd be tempted to require dyed lumber if that was a thing.